pkg/targets/adapter/twiliotarget/adapter.go

```diff
@@ -85,9 +97,10 @@ func (a *twilioAdapter) dispatch(event cloudevents.Event) cloudevents.Result {
 	}
 
 	if _, err := a.client.Messages.SendMessage(sms.From, sms.To, sms.Message, sms.MediaURLs); err != nil {
-		return fmt.Errorf("error sending message: %w", err)
+		return a.replier.Error(&event, targetce.ErrorCodeAdapterProcess, err, nil)
 	}
 
-	a.logger.Infof("Sent message: %s", event.String())
-	return cloudevents.ResultACK
+	info := fmt.Sprintf("message sent to: %s", sms.To)
+	a.logger.Info(info)
```

Comment:
`Debug(...)` sounds more appropriate here. We should log only errors by default, successes only for troubleshooting.

Reply:
fixed!